pkgs/servers/x11/xorg/default.nix

```diff
@@ -858,11 +858,11 @@ lib.makeScope newScope (self: with self; {
   # THIS IS A GENERATED FILE.  DO NOT EDIT!
   libX11 = callPackage ({ stdenv, pkg-config, fetchurl, xorgproto, libxcb, xtrans }: stdenv.mkDerivation {
     pname = "libX11";
-    version = "1.7.2";
+    version = "1.8.1";
     builder = ./builder.sh;
     src = fetchurl {
       url = "mirror://xorg/individual/lib/libX11-1.7.2.tar.bz2";
```

Comment:
This seems wrong

Reply:
We should get rid of the generator now that katamari tarballs are not a thing.